sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -487,20 +487,28 @@ string_class ProgramManager::getProgramBuildLog(const RT::PiProgram &Program,
   string_class Log = "The program was built for " +
                      std::to_string(PIDevices.size()) + " devices";
   for (RT::PiDevice &Device : PIDevices) {
+    std::string DeviceBuildInfoString = "";
```

Comment:
```suggestion
    std::string DeviceBuildInfoString;
```
since this is the default value anyway.